cirrus/server/pyproject.toml

```diff
@@ -10,7 +10,7 @@ ruff = "^0.0.254"
 fastapi = "^0.97.0"
 uvicorn = "^0.20.0"
 pytest = "^7.2.2"
-httpx = "^0.23.3"
+httpx = "^0.24.1"
```

Comment:
@yashika-khurana we don't actually need httpx in cirrus do we?

Reply:
for `TestClient` we need httpx, here are the [docs](https://fastapi.tiangolo.com/tutorial/testing/)